modules/kn-service-update.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-update_{context}"]
+= Updating serverless applications
+
+The `kn service update` command is useful for interactive sessions on the command line where a user builds up a service incrementally. In contrast to the `kn service apply` command, when using the `kn service update` command you only have to specify the changes that you want to update, rather than the full configuration for the Knative service.
```

Comment:
I like the second sentence better where you use the _you_ structure. Can you change the first sentence to match?
Something like: _You can use the `kn service update` command for interactive sessions on the command line as you build up a service incrementally._

Reply:
I wonder, too, if you should swap the sentences around, to define what `kn service update` does, then show how it can be _especially useful__?
Something like:
_You can use the `kn service update` command to make specific changes to a service. In contrast to the kn service apply command, when using the kn service update command you only have to specify the changes that you want to update, rather than the full configuration for the Knative service. (I love this sentence, really points out the difference!) The `kn service apply` command is especially useful..._

Reply:
Great suggestion, thank you!